### DISCUSSION TITLE
how to resize and entire drawing to scale?

### DISCUSSION THREAD

**Discussion by enterausernameagain** [jgraph/drawio-desktop#1047] [General]
Hi,

I made an elaborate sketch to be printed on a 24 x 36 architectural size paper. It's a bit larger than expected, is there anyway for me to shrink the entire drawing to scale?

**Comment by davidjgraph** [comment]
There's a zoom option in advanced export.

**Comment by enterausernameagain** [comment]
Will this allow me to shrink the drawing enough to fit my dimensions
without quality loss? I don't have to shrink it by much I'm bleeding over
into page 2 by a hair.
